docs/kyma/docs/042-gardener-kyma-deployment.md

```diff
@@ -0,0 +1,79 @@
+# Kyma on Gardener
+
+## Overview
+
+This walkthrough is designed to guide you step-by-step through the process of installing Kyma on Gardener. See the official [Gardener Github](https://github.com/gardener/gardener) page for more information regarding the project.
```

Comment:
This walkthrough is designed to guide you step-by-step through the process of installing Kyma on Gardener. ->
This Getting Started guide shows developers how to install Kyma on a [Gardener](https://github.com/gardener/gardener) cluster.